modules/gdscript/gdscript_warning.cpp

```diff
@@ -94,6 +94,9 @@ String GDScriptWarning::get_message() const {
 				return vformat(R"*(%s "%s()" has no static return type.)*", symbols[0], symbols[1]);
 			}
 			return vformat(R"(%s "%s" has no static type.)", symbols[0], symbols[1]);
+		case INFERRED_DECLARATION:
+			CHECK_SYMBOLS(2);
+			return vformat(R"(%s "%s" has an inferred static type.)", symbols[0], symbols[1]);
```

Comment:
I thought for a while about this warning message. At first, I didn't like it, but I think I've decided it's fine.

Reply:
Maybe so?

```suggestion
			return vformat(R"(%s "%s" has an implicitly inferred static type.)", symbols[0], symbols[1]);
```

Reply:
Yeah! I think including that it's implicit tells the user why they're getting the warning.